dns/zones/terraform.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+  required_version = "~> 0.15.5"
```

Comment:
Can we update terraform and the azurerm module to the latest versions?

Reply:
updated tf and azurerm to latest versions